static/canjs.js

```diff
@@ -95,7 +95,7 @@ function init() {
 				window.history.pushState(null, null, href);
 				navigate(href);
 			},
-			pathPrefix: window.pathPrefix,
+			pathPrefix: '/doc',
```

Comment:
Hard setting this to the correct value for the live site means changing the testing environment to match. I am open to other ideas, but this is the cleanest option I found.